scripts/weblate/weblate_locks_notify_community.py

```diff
@@ -17,18 +17,28 @@
 projects['playbooks']='playbooks/webapp'
 projects['glossary']='mattermost/glossary'
 
-w=wlc.Weblate(apikey.myAPIKey)
+mention={}
+mention['mobile']='cc: @guillermo.vaya,@zef.hemel'
+mention['server']='cc: @guillermo.vaya,@zef.hemel'
+mention['webapp']='cc: @guillermo.vaya,@zef.hemel'
+mention['focalboard']='cc: @jesus.espino,@scott.bishel'
+mention['playbooks']='cc: @jesse.hallam,@caleb.roseland'
```

Comment:
One more Dev owner can be added: Desktop App issues are owned by @devinbinnie

Reply:
added!